src/xdocs/google_style.xml

```diff
@@ -21,10 +21,10 @@
                 <p>
                     This coverage report was created for
                     <a href="http://google.github.io/styleguide/javaguide.html">
-                        Google Java Style</a>(
-                    <a href="styleguides/google-java-style-20170228.html">
-                          cached page</a>)
-                    , version of 28 February 2017
+                        Google Java Style</a>(<a
```

Comment:
please restore formatting here, I like `a` to start a line.

Reply:
In source view I'd agree.
But here I cared more about the result.
Your version:
![grafik](https://user-images.githubusercontent.com/325196/57289505-cc485b00-70bb-11e9-9f1a-9f195223fed2.png)
My version:
![grafik](https://user-images.githubusercontent.com/325196/57289475-bd61a880-70bb-11e9-8d9d-0703514cce5a.png)
Like it was before, there are spurious spaces that imho shouldn't be there, which is why I moved the beginning of the `<a>` and the `comma` to the end of the previous line.
Don't you think the result is more important to be nice here?

Reply:
Thanks, for explanation, let's keep your update